src/main.js

```diff
@@ -59,7 +59,7 @@ library.add(faCoffee, faCopy, faCogs, faHistory, faPlay, faImage, faLayerGroup,
 ),
 
 Vue.use(ClientTable, {}, false, 'bootstrap4', 'default');
-Vue.use(ServerTable, {}, true, 'bootstrap4', 'default');
```

Comment:
Actually the vue-table **never used Vuex** because the registration was wrong.

In this commit https://github.com/eubr-bigsea/citrus/commit/503cb4b6935adb933ae5d16c8609936c2457c072#diff-1c90ff38a08209f9ebd4d05d1e43358e

I fixed the registration of vue-table. The parameters were being passed in the wrong way.

But I set the **useVuex** option to true, which led to some bugs. (#68)

So I made this PR to fix that.

About the behavior of persist filters between requests, it still works. When set to false, it uses localStorage instead.

But If you still want to use vuex, just let me know and I'll make it work. :smiley: